packages/ipfs-http-client/src/add/index.js

```diff
@@ -32,16 +32,18 @@ module.exports = configure(({ ky }) => {
     if (options.preload != null) searchParams.set('preload', options.preload)
     if (options.fileImportConcurrency != null) searchParams.set('file-import-concurrency', options.fileImportConcurrency)
     if (options.blockWriteConcurrency != null) searchParams.set('block-write-concurrency', options.blockWriteConcurrency)
-
+    
+    const formData = await toFormData(input)
```

Comment:
@hugomrdias @achingbrain Do you have any stylistic objections to building `formData` in a separate line like this?